packages/accounts/src/msca/account-loupe/decorator.ts

```diff
@@ -47,7 +47,7 @@ export type AccountLoupeActions<
   /// @notice Gets an array of all installed plugins
   /// @return The addresses of all installed plugins
   getInstalledPlugins(
-    args: GetAccountParameter<TAccount>
+    args?: GetAccountParameter<TAccount>
```

Comment:
we shouldn't do this. it won't give the user the right feedback if the account isn't hoisted. Instead you may want to do what we do for `MultiOwnerPluginActions`.

We need to enforce `args` to not be undefined if `TAccount is undefined`